d3a_interface/sim_results/cumulative_grid_trades.py

```diff
@@ -210,28 +220,6 @@ def _accumulate_area_trades(cls, area, parent, flattened_area_core_stats_dict,
 
         return accumulated_trades
 
-    @classmethod
-    def generate_cumulative_grid_trades_for_all_areas(cls, accumulated_trades, area,
-                                                      parent, results):
-        if area['children'] == []:
-            return results
-
-        results[area['uuid']] = [CumulativeGridTrades.generate_area_cumulative_trade_redis(
-                child, area, accumulated_trades
-            )
-            for child in area['children']
-            if child['name'] in accumulated_trades
-        ]
-        if parent is not None:
-            results[area['uuid']].append(CumulativeGridTrades._external_trade_entries(
-                area, accumulated_trades))
-
-        for child in area['children']:
-            results = cls.generate_cumulative_grid_trades_for_all_areas(
-                accumulated_trades, child, area, results
-            )
-        return results
-
```

Comment:
Nice, so now we save this effort. It is now done in `generate_cumulative_grid_trades_target_area` only for the target area on the d3a-web side, right?